beacon-chain/operations/attestations/prepare_forkchoice.go

```diff
@@ -39,7 +39,11 @@ func (s *Service) prepareForkChoiceAtts() {
 			if err := s.batchForkChoiceAtts(s.ctx); err != nil {
 				log.WithError(err).Error("Could not prepare attestations for fork choice")
 			}
-			log.WithField("latency", time.Since(t).Milliseconds()).Debug("batched forkchoice attestations")
+			if slots.TimeIntoSlot(s.genesisTime) < intervals[1] {
```

Comment:
panic if `intervals` is length 0. Probably better to do a length check

Reply:
Preston had the same comment, but interval is a hardcoded configuration flag, this can never be 0

Reply:
solved by converting to array